plugin-template

```diff
@@ -44,10 +45,24 @@ DEFAULT_SETTINGS = {
     'pulpcore_pip_version_specifier': None,
     'pypi_username': None,
     'stable_branch': None,
+    's3_settings': {
+        'aws_access_key_id': '"AKIAIT2Z5TDYPX3ARJBA"',
```

Comment:
Aren't we hardcoding this value in the test scripts? I'm thinking we should just hardcode all of these s3 settings as well as I doubt plugin writers will need to tweak them.